pkg/catalog/types.go

```diff
@@ -7,9 +7,9 @@ import (
 
 // ComponentType is the main struct for catalog components
 type ComponentType struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              ComponentSpec `json:"spec,omitempty"`
+	metav1.TypeMeta		`json:",inline"`
```

Comment:
Formatting is not proper here. It's an issue in other files as well. Travis checks are failing for the same reason:
```sh
$ make validate
./scripts/check-gofmt.sh
gofmt ERROR - These files are not formated by gofmt:
./pkg/odo/cli/catalog/util/util.go
./pkg/catalog/types.go
./pkg/machineoutput/services.go
./pkg/machineoutput/types.go
./pkg/service/types.go
Makefile:65: recipe for target 'gofmt' failed
make: *** [gofmt] Error 1
The command "make validate" exited with 2.

```
You can fix this with the go fmt tool.

Reply:
Thanks for pointing that out. Changes made.